docs/getting-started.md

```diff
@@ -26,6 +26,18 @@ A quick start guide to get KubeVirt up and running inside Vagrant.
     sudo dnf install golang
 ```
 
+**Note:** Some code within k8s.io/client-go and k8s.io/apimachinery uses
+features from the Go standard libaries introduced in version 1.8. This affects
+Fedora 25 as it ships with Go 1.7.5.
+
+A helpful tool to dynamically manage multiple versions of Go is
+[gimme](https://github.com/travis-ci/gimme)
+
+```bash
+    eval "$(curl -sL \
+    https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | \
+    GIMME_GO_VERSION=stable bash)"
+```
```

Comment:
This is the exact same command listed on the gimme README.md landing page, which users will find and execute anyways. The link in question of course points to openly available content hosted directly on github. Either you trust the tool or you don't.